benchmarks/src/jmh/java/io/micrometer/benchmark/tracer/BraveTracerBenchmark.java

```diff
@@ -0,0 +1,151 @@
+            this.bridgeContext = new BraveCurrentTraceContext(this.braveCurrentTraceContext);
+            this.tracing = Tracing.newBuilder()
+                .currentTraceContext(this.braveCurrentTraceContext)
+                .sampler(Sampler.NEVER_SAMPLE)
+                .addSpanHandler(SpanHandler.NOOP)
+                .build();
+            this.tracing.setNoop(true);
+            this.braveTracer = this.tracing.tracer();
+            this.tracer = new BraveTracer(this.braveTracer, this.bridgeContext, new BraveBaggageManager());
+        }
+
+        @TearDown
+        public void close() {
+            this.tracing.close();
+        }
+
+    }
+
+    @Benchmark
+    public void micrometerTracing(MicrometerTracingState state, Blackhole blackhole) {
+        micrometerTracing(state.tracer, state.childSpanCount, blackhole);
+    }
+
+    @Benchmark
+    public void micrometerTracingNewSpan(MicrometerTracingState state, Blackhole blackhole) {
+        micrometerTracingNewSpan(state.tracer, blackhole);
+    }
+
+    @Benchmark
+    public void micrometerTracingWithScope(MicrometerTracingState state, Blackhole blackhole) {
+        micrometerTracingWithScope(state.tracer, state.childSpanCount, blackhole);
+    }
+
+    @State(Scope.Benchmark)
+    public static class BraveState {
+
+        @Param({ "5" })
+        public int childSpanCount;
+
+        ThreadLocalCurrentTraceContext braveCurrentTraceContext;
+
+        Tracing tracing;
+
+        brave.Tracer tracer;
+
+        @Setup
+        public void setup() {
+            this.braveCurrentTraceContext = ThreadLocalCurrentTraceContext.newBuilder().build();
+            this.tracing = Tracing.newBuilder()
+                .currentTraceContext(this.braveCurrentTraceContext)
+                .sampler(Sampler.NEVER_SAMPLE)
+                .addSpanHandler(SpanHandler.NOOP)
+                .build();
+            this.tracing.setNoop(true);
+            this.tracer = this.tracing.tracer();
+        }
+
+        @TearDown
+        public void close() {
+            this.tracing.close();
+        }
+
+    }
+
+    @Benchmark
+    public void braveTracing(BraveState state, Blackhole blackhole) {
+        brave.Span parentSpan = state.tracer.nextSpan().name("parent-span").start();
+        TraceContextOrSamplingFlags traceContext = TraceContextOrSamplingFlags.create(parentSpan.context());
+        for (int i = 0; i < state.childSpanCount; i++) {
+            brave.Span span = state.tracer.nextSpan(traceContext).name("new-span" + i);
+            span.start().tag("key", "value").annotate("event").finish();
+        }
+        parentSpan.finish();
+        blackhole.consume(parentSpan);
+    }
+
+    @Benchmark
+    public void braveTracingNewSpan(BraveState state, Blackhole blackhole) {
+        brave.Span span = state.tracer.nextSpan().name("child-span").start();
+        span.finish();
+        blackhole.consume(span);
```

Comment:
In this case, I think we can just return the Span from the method and this will be consumed by the harness.